index.html

```diff
@@ -75,6 +75,130 @@ <h3 class="lets-connect">Let's connect</h3>
       <div class="next-button">
       </div>
     </section>
+    <section class="works">
+      <h1 class="title">My Recent Works</h1>
+      <img src="/resources/Indicator.svg" alt="Indicator">
+
+      <div class="project-card">
+        <div class="project-image project1"></div>
+        <div class="project-info">
+          <h3 class="project-title">Multi-Post Stories
+            <br> Gain+Glory
+          </h3>
+          <ul class="project-languages">
+            <li>Ruby on rails</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <div class="project-button button">See
+            Project</div>
+        </div>
+      </div>
+      <div class="project-card">
+        <div class="project-image project2"></div>
+        <div class="project-info">
+          <h3 class="project-title">Multi-Post Stories
+            <br> Gain+Glory
+          </h3>
+          <ul class="project-languages">
+            <li>Ruby on rails</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <div class="project-button button">See
+            Project</div>
+        </div>
+      </div>
+      <div class="project-card">
+        <div class="project-image project3"></div>
+        <div class="project-info">
+          <h3 class="project-title">Multi-Post Stories
+            <br> Gain+Glory
+          </h3>
+          <ul class="project-languages">
+            <li>Ruby on rails</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <div class="project-button button">See
+            Project</div>
+        </div>
+      </div>
+      <div class="project-card">
+        <div class="project-image project4"></div>
+        <div class="project-info">
+          <h3 class="project-title">Multi-Post Stories
+            <br> Gain+Glory
+          </h3>
+          <ul class="project-languages">
+            <li>Ruby on rails</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <div class="project-button button">See
+            Project</div>
+        </div>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="text-section">
+        <h1 class="title">About me</h1>
+        <p class="supporting-text">I can help you build a
+          product , feature or website Look
+          through some of my work and experience! If you
+          like what you
+          see and have a project you need coded,
+          don’t hesitate to contact me.</p>
+        <div class="resume-button button">Get my Resume
+        </div>
```

Comment:
- [ ] [Required] Nice job in implementing the Pressed, Hover, and Active effect here. Please can we change the `<div> </div>` element to `<button> </button>` element?.

Reply:
Yes, sure!!
I changed all the design button that had `<div></div>` to `<button></button>` including the ones in the **Works** section.
Best,